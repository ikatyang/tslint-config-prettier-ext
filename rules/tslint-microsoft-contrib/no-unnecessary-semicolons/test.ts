while (true) {
  console.log('semi');
};
