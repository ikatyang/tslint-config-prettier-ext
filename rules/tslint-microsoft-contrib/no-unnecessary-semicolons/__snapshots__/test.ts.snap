// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (no-unnecessary-semicolons) /rules/tslint-microsoft-contrib/no-unnecessary-semicolons/test.ts[3, 2]: unnecessary semi-colon

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
while (true) {
  console.log('semi');
};

======
while (true) {
  console.log(\\"semi\\");
}

>>>>>> after
"
`;
